docs/framework/react/guide/code-splitting.md

```diff
@@ -296,7 +296,7 @@ If you're not using the file-based routing system, you can still manually split
 - `posts.lazy.tsx`
 
   ```tsx
-  export const Route = createLazyRoute({
+  export const Route = createLazyRoute('/posts')({
```

Comment:
Wow
I have this structure:

_main.tsx_
```tsx
export const postsRoute = createRoute({
  getParentRoute: () => rootRoute,
  path: '/swap',
}).lazy(() => import('./views/swap.lazy').then((d) => d.Route))
```

_views/SwapView.tsx_
```tsx
export const Route = createLazyRoute('/swap')({
  component: SwapView,
})
```

And it works well for me

Reply:
> example has
> 
> ```tsx
> createLazyRoute('/')({
> ```
> 
> https://github.com/TanStack/router/blob/main/examples/react/basic/src/posts.lazy.tsx#L5
> 
> which one is correct?

Its meant to be the equivalent of `createLazyFileRoute` but for code-splitting out the `component`,`notFoundComponent`,`errorComponent` as a single bunch, and for returning the typed hooks for that route.

The example has it used incorrectly. It doesn't throw any errors in the example, since the actual route's methods aren't being consumed. If used correctly like this, you'd see the error.
```tsx
export const Route = createLazyRoute('/posts')({
  component: PostsComponent,
});

function PostsComponent() {
  const posts = Route.useLoaderData();
  // ...
}
```